src/app/public/modules/flyout/flyout-media-query.service.spec.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  inject,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  Subscription
+} from 'rxjs/Subscription';
+
+import {
+  SkyMediaBreakpoints,
+  SkyWindowRefService
+} from '@skyux/core';
+
+import {
+  SkyFlyoutAdapterService
+} from './flyout-adapter.service';
+
+import {
+  SkyFlyoutMediaQueryService
+} from './flyout-media-query.service';
+
+describe('Flyout media query service', () => {
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        SkyFlyoutAdapterService,
+        SkyFlyoutMediaQueryService,
+        SkyWindowRefService
```

Comment:
Should we create a test module to follow our more recently adopted pattern?

Reply:
I would say no. We already have one for the component tests. But for the service we don't have fixtures and such. I think importing a lot of stuff we don't need isn't really needed when we are just adding three providers here.

Reply:
But.. wouldn't you import two less items if you had a module :) ?

It's not a deal breaker for me!